src/chatClient.cpp

```diff
@@ -2437,7 +2441,18 @@ void ContactList::syncWithApi(mega::MegaUserList& users)
 
 void ContactList::onUserAddRemove(mega::MegaUser& user)
 {
-    addUserFromApi(user);
+    if (user.getVisibility() == ::mega::MegaUser::VISIBILITY_INACTIVE)
```

Comment:
Are you sure you don't want to add an INACTIVE user to the contactlist? I remember we wanted to have all users in the contactlist, and only show the ones that are not INACTIVE in the GUI.